cvat/apps/engine/media_extractors.py

```diff
@@ -90,13 +90,26 @@ def _get_preview(obj):
         else:
             preview = obj
         preview.thumbnail(PREVIEW_SIZE)
+        orientation = preview._getexif().get(274, 1) if preview._getexif() else 1
```

Comment:
Looks like _getexif() is a private method. Using such methods outside is not good practice. 
What these magic numbers (274, 1) means is not obvious.

Reply:
`_getexif()` is a convenient method to get EXIF information from PIL Image. 274 is the Orientation key, it can be gotten via 
```(python)
(orientation_key,_), = filter(lambda kv: kv[1]=='Orientation', ExifTags.TAGS.items())
```
But it looks messy. Probably I will add cont with some description.